b2share/modules/records/serializers/schemas/json.py

```diff
@@ -43,11 +43,15 @@ def filter_internal(self, data):
         if '_files' in data['metadata']:
             del data['metadata']['_files']
         if '_pid' in data['metadata']:
+            epic_pids = [p for p in data['metadata']['_pid']
+                         if p.get('type') == 'handle_pid']
+            if len(epic_pids) > 0:
+                data['metadata']['PID'] = epic_pids[0].get('value')
```

Comment:
That's how it is in v1, but you might be right. Issue: https://github.com/EUDAT-B2SHARE/b2share/issues/1015